bluemira/equilibria/flux_surfaces.py

```diff
@@ -123,6 +123,43 @@ def connection_length(self, eq):
         """
         return np.sum(self._dl(eq))
 
+    def poloidal_angle(self, eq, x_strike, z_strike, gamma):
+        """
+        From glancing angle (gamma) to poloidal angle.
+
+        Parameters
+        ----------
+        eq: Equilibrium
+            Equilibrium from which to take magnetic fields
+            at the strike pointcalculate the flux expansion
+        x_strike: float
+            x coordinate of strike point
+        z_strike: float
+            z coordinate of strike point
```

Comment:
This point should be on the `OpenFluxSurface` but there is no check to that effect. I could ask for the poloidal angle of an `OpenFluxSurface` at a point nowhere near the flux surface and get a result from this.

In fact, this can be seen by the fact that this method is effectively a totally static method (no self used anywhere).

The question is then: is this simply a function that is independent from a flux surface? I'm leaning towards yes. In which case, this might better live in `equilibria/find.py` or indeed here, but as a stand-alone function.

Reply:
> This point should be on the `OpenFluxSurface` but there is no check to that effect. I could ask for the poloidal angle of an `OpenFluxSurface` at a point nowhere near the flux surface and get a result from this.
> 
> In fact, this can be seen by the fact that this method is effectively a totally static method (no self used anywhere).
> 
> The question is then: is this simply a function that is independent from a flux surface? I'm leaning towards yes. In which case, this might better live in `equilibria/find.py` or indeed here, but as a stand-alone function.

I vote for stadalone version and keeping the function here.

Reply:
Yep, sounds good